src/java_bytecode/java_bytecode_instrument.cpp

```diff
@@ -100,7 +100,7 @@ codet java_bytecode_instrumentt::throw_exception(
     generate_class_stub(
       exc_name,
       symbol_table,
-      get_message_handler());
+      message_handler);
```

Comment:
This seems like a back-step (and I got different auto-resolution) so you might want to check this.